scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -596,7 +599,7 @@ function Install-Packages(
     if ($Update) {
         Write-LogInformation
     }
-    else {
+    elseif (-not (Test-Path Variable:\calledFromInstall)) {
```

Comment:
@arsing  This will imho not work if somebody tries to run the cmdlets multiple times from once loaded module. Imho we should set true/false as part of try/finally in the install and test that here.

Reply:
It will work. Variables are destroyed when their scope exits.

```powershell
$ function Foo { $bar = $true; Test-Path Variable:\bar }; Test-Path Variable:\bar; Foo; Test-Path Variable:\bar

False
True
False
```

Reply:
My bad, you were right. I was thinking these had different scope.